release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -0,0 +1,573 @@
+$ for clusterrolebinding in cluster-status-binding discovery system:basic-user system:discovery system:openshift:discovery ;
+do
+### Find the index of unauthenticated group in list of subjects
+index=$(oc get clusterrolebinding ${clusterrolebinding} -o json | jq 'select(.subjects!=null) | .subjects | map(.name=="system:unauthenticated") | index(true)');
+### Remove the element at index from subjects array
+oc patch clusterrolebinding ${clusterrolebinding} --type=json --patch "[{'op': 'remove','path': '/subjects/$index'}]";
+done
+----
++
+This script removes unauthenticated subjects from the following cluster role bindings:
++
+--
+** `cluster-status-binding`
+** `discovery`
+** `system:basic-user`
+** `system:discovery`
+** `system:openshift:discovery`
+--
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1821771[*BZ#1821771*])
+
+// TODO: This known issue should carry forward to 4.9 and beyond!
+* The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
+
+[id="ocp-4-9-asynchronous-errata-updates"]
+== Asynchronous errata updates
+
+Security, bug fix, and enhancement updates for {product-title} 4.9 are released as asynchronous errata through the Red Hat Network. All {product-title} 4.9 errata is https://access.redhat.com/downloads/content/290/[available on the Red Hat Customer Portal]. See the https://access.redhat.com/support/policy/updates/openshift[{product-title} Life Cycle] for more information about asynchronous errata.
+
+Red Hat Customer Portal users can enable errata notifications in the account settings for Red Hat Subscription Management (RHSM). When errata notifications are enabled, users are notified via email whenever new errata relevant to their registered systems are released.
+
+[NOTE]
+====
+Red Hat Customer Portal user accounts must have systems registered and consuming {product-title} entitlements for {product-title} errata notification emails to generate.
+====
+
+This section will continue to be updated over time to provide notes on enhancements and bug fixes for future asynchronous errata releases of {product-title} 4.9. Versioned asynchronous releases, for example with the form {product-title} 4.9.z, will be detailed in subsections. In addition, releases in which the errata text cannot fit in the space provided by the advisory will be detailed in subsections that follow.
+
+[IMPORTANT]
+====
+For any {product-title} release, always review the instructions on xref:../updating/updating-cluster.adoc#updating-cluster[updating your cluster] properly.
+====
+
+[id="ocp-4-9-0-ga"]
+=== #RHSA-2021:TBD1# - {product-title} #4.9.0# image release, bug fix, and security update advisory
```

Comment:
This whole section has a bunch of weird highlighting that I've never seen before. Probably intentional, but just wanted to check!

Reply:
Yeah making it ugly so we see it later :+1: